internal/model/user.go

```diff
@@ -32,22 +34,26 @@ func NewUser(id string, name string) User {
 	}
 }
 
+// HasActiveBuilds checks if current user has any active builds
```

Comment:
And what does it return?

"HasActiveBuilds checks if current user has any active builds. If so true is returned, otherwise false"